index.html

```diff
@@ -0,0 +1,154 @@
+	<script src="scripts/adopt-dog.js"></script>
+</head>
+
+<body onload="reset()" onunload="reset()">
+	<header>
+		<div class="head-checkout">
+			<div class="logo">
+				<figure>
+					<img src="images/adoptadog-logo.svg" alt="dog LOGO">
+					<figcaption>Seattle’s premiere dog adoption agency</figcaption>
+				</figure>
+			</div>
+			<div class="checkout">
+				<a href="#"><img src="images/cart-icon.svg" alt="checkout image"></a><em id="cost">$0</em>
+				<p>
+				<button onclick='checkOut()' type="button">Checkout</button>
+			</div>
+		</div>  <!-- for "head-checkout -->
+		<nav class="clear">
+			<a href="index.html">Home</a> /
+			<a href="dogs.html">Dogs</a> / 
+			<a href="blog.html">Blog</a>
+		</nav>
+	</header>
+
+<main>
+	<section>
+		<img src="images/home-intro.jpg" alt="home introduction image" class="intro-img">
+		<div class="intro">
+			<h2>Adopt A Dog </h2>
+			<p>Iduciendisite quo magnatem iuntum quid quaest ea am,
+			tenderumet adis dolenem quidustrum fuga. Faceaquae estioria
+			derum recuptatur, cum volore, undipsa doloreium hillupta aut
+			es ut alitatuscit ommossum haritatur arum qui officae videbiti
+			corporeium faccull oribus es quidignis ipietus explam sus
+			am aut amet ant fugiatum, utem non reptat ad magnienecto
+			volorent fugia nonsedi gendand itatquos as magnimpor aut
+			molor autatia ad est officimus accae et omnia cus, od qui ate</p>
+		</div>
+
+		<div class="clear flex-container">
+			<div>
+				<img onclick='dogObj({name:"murphy",breed:"beagle",cost:100.10})' src="images/murphy-card.jpg" alt="murphy dog image">
+				<div class="dog">
+					<p>Murphy</p>
+					<p><strong>Cost to Adopt:</strong> $100.10</p>
+					<p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+					<button onclick='buyIt(100.10)' type="button">Adopt</button>
+				</div>
+			</div>
+
+			<div>
+				<img onclick='dogObj({name:"poppy",breed:"bullmastiff",cost:100.20})' src="images/poppy-card.jpg" alt="poppy dog image">
+				<div class="dog">
+					<p>Poppy </p>
+					<p><strong>Cost to Adopt:</strong> $100.20</p>
+					<p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+					<button onclick='buyIt(100.20)' type="button">Adopt</button>
+				</div>
+			</div>
+
+			<div>
+				<img onclick='dogObj({name:"jack",breed:"bloodhound",cost:100.30})' src="images/jack-card.jpg" alt="jack dog image">
+				<div class="dog">
+					<p>Jack</p>
+					<p><strong>Cost to Adopt:</strong> $100.30</p>
+					<p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+					<button onclick='buyIt(100.30)' type="button">Adopt</button>
+				</div>
+			</div>
+			
+			<div>
+				<img onclick='dogObj({name:"duffy",breed:"akita",cost:100.40})' src="images/duffy-card.jpg" alt="duffy dog image">
```

Comment:
When using JS in HTML it's good to wrap the attribute in double quotes, and the things inside in single quotes

```
<img onclick="dogObj({name:'duffy', breed:'akita', cost:100.40})" src="images/duffy-card.jpg" alt="duffy dog image">
```